x/ccv/provider/types/proposal.go

```diff
@@ -204,6 +204,7 @@ func (sccp *ConsumerRemovalProposal) ValidateBasic() error {
 }
 
 // NewEquivocationProposal creates a new equivocation proposal.
+// [DEPRECATED]: do not use
```

Comment:
I guess this message is related to this https://github.com/cosmos/interchain-security/pull/1394? I would add a message on why it's deprecated and why it's still here nevertheless.

I was wondering, do we have to keep this piece of code and the corresponding equivocation proposal protos or could we get rid of them when we release a new _MAJOR_ version?

Reply:
Added more info on why it's deprecated. Responded below why it's not simple to remove it.